crypto/libtomcrypt/Kconfig

```diff
@@ -54,7 +54,6 @@ endif # LIBTOMCRYPT_LTCRYPT
 
 config LIBTOMCRYPT_HASHSUM
 	tristate "LibTomCrypt hashsum"
-	depends on SCHED_ATEXIT
```

Comment:
why removing `depends on SCHED_ATEXIT` ?

Reply:
this option is removed long time ago.

Reply:
hmm so why the CI fails? :-(